### ISSUE TITLE
problem with derecho modules and running AIS CISM simulations

### ISSUE THREAD

**Issue by gunterl** [ESCOMP/CISM#68]
When trying to run Antarctic runs using the file derecho-intel-modules from the build directory, CISM kept failing thinking only 1 task was being requested while I requested 384. When I revert using an older version of derecho-intel-modules, CISM runs recognizing all requested tasks. 

**The current derecho-intel-modules (which makes CISM fail) loads the following:**
module load ncarenv/23.09
module load intel/2023.2.1
module load ncarcompilers/1.0.0
module load cray-mpich/8.1.27
module load mkl/2023.2.0
module load netcdf/4.9.2
module load cmake/3.26.3

**The modules I am loading now with successful CISM run are:**
module load ncarenv/23.09
module load intel-oneapi/2023.2.1
module load intel-mpi/2021.10.0
module load mkl

**Comment by whlipscomb** [comment]
@gunterl, this error is familiar to me. The last time I saw it, it was connected to having loaded the wrong modules. But lately, everything has been working for me, as long as I build the code with the 7 modules listed in the current derecho-intel-modules, and then source that same file in my run script before launching the run.

Did you build your executable with these new modules? I think I've run into issues when I've tried to run an older executable.
